packages/es-components/src/components/containers/modal/Modal.js

```diff
@@ -105,31 +103,33 @@ function Modal(props) {
     ...other
   } = props;
 
-  const ModalDialog = getModalBySize(size);
   const ariaId = useUniqueId(other.id);
 
+  const shouldCloseOnOverlayClick = backdrop !== 'static' && backdrop !== false;
```

Comment:
This doesn't seem to be working as it used to. Clicking outside the modal doesn't close it anymore, and setting the `backdrop` prop to "static", "true", or "false" doesn't seem to do anything.
If the backdrop is set to "static", clicking the backdrop area shouldn't close the modal.
If "false" there should be no opaque backdrop.
If "true" (default) it should have the opaque backdrop but close when clicking outside the modal.

Reply:
This should be fixed now.

Reply:
"Static" and "true" work, but false still shows the backdrop.